elastic/kibana
# Allow task manager health stats to be logged as info messages

In this PR, I'm adding a new setting (`xpack.task_manager.monitored_stats_health_verbose_log.level`) that allows the task manager monitoring stats to be verbosely logged at info level instead of warning.

The two supported values are:
- debug (default)
- info

This will help debug SDHs on Cloud where we won't want to turn on debug level on the entire cluster but would still like to see the task manager monitored stats over time.

## Cloud allow-list PR

https://github.com/elastic/cloud/pull/109563

## To verify
1. Set the following two configuration options:
```
xpack.task_manager.monitored_stats_health_verbose_log.enabled: true
xpack.task_manager.monitored_stats_health_verbose_log.level: info
```
2. Startup Kibana
3. Notice `Latest Monitored Stats:` are logged at info level
4. Remove `xpack.task_manager.monitored_stats_health_verbose_log.level` configuration
5. Add the following configuration
```
logging:
  loggers:
    - name: plugins.taskManager
      level: debug
```
6. Restart Kibana
7. Notice `Latest Monitored Stats:` are logged at debug level (as usual)
